br/tests/lightning_duplicate_resolution_replace/config.toml

```diff
@@ -0,0 +1,17 @@
+[lightning]
+task-info-schema-name = 'lightning_task_info'
+
+[tikv-importer]
+backend = 'local'
+duplicate-resolution = 'replace'
+add-index-by-sql = false
+
+[checkpoint]
+enable = false
+
+[mydumper]
+batch-size = 1
+# ensure each file is its own engine to facilitate cross-engine detection.
```

Comment:
Maybe we do not need to add a test for cross-engine, because the we use the existing logic of collecting and resolving the conflicts:

https://github.com/pingcap/tidb/blob/7e66761047b01b62c4097892be14555f83a3c3b3/br/pkg/lightning/importer/table_import.go#L991-L1036

The three main functions are: `dupeController.CollectLocalDuplicateRows`, `dupeController.CollectRemoteDuplicateRows`, `dupeController.ResolveDuplicateRows`. I fully add the algorithms for `replace` in `dupeController.ResolveDuplicateRows`, so the logic for cross-engine is the same for the `replace` algorithm, and we do not need to test it.